composer.json

```diff
@@ -21,6 +21,7 @@
     },
     "require-dev": {
         "ergebnis/composer-normalize": "^2.28",
+        "rector/rector": "^0.14.6",
```

Comment:
I suggest to use a distinct version (like "0.14.6") and update explicitely. I made the experience that patch updates returns an error where there wasn't one in the previous patch version. This then breaks the current PR (which should not).